### PR TITLE
Home

### PR DESCRIPTION
**Pull request by sadathshariff**
Created a Basic Layout for all the Pages,
Home Pages Contains the Following Components
1. Modal
2. Card
3. Comment
4. Chips
5. SideNav
6. FollowBar
7. BottomNav

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
Evrything looks good, nice arrangement and component breakdown

File: src/components/BottomNav/BottomNav.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <ul className={`${styles.bottomNav_container} list flex justify-sb`}>
+        <li>
+          <Link to="/feed">
+            {/* <div className={`${styles.sidenav_item}`}></div> */}
+            <BiHomeSmile size={27} />
+          </Link>
+        </li>
+        <li>
+          <Link to="/explore">
+            {/* <div className={`${styles.sidenav_item}`}>
```
Comment (nishasen) [inline src/components/BottomNav/BottomNav.jsx]: remove the commented code

**Review by nishasen — APPROVED** [review]
Looks good, and clean code

File: src/components/BottomNav/BottomNav.jsx (hunk 1 above)
Comment (sadathshariff) [reply src/components/BottomNav/BottomNav.jsx]: Thanks for pointing it out, Nisha

**Pull request merged**